pkg/pillar/hypervisor/kvm.go

```diff
@@ -406,6 +406,9 @@ func (ctx kvmContext) Setup(status types.DomainStatus, config types.DomainConfig
 		dmArgs = append(dmArgs, ctx.dmCPUArgs...)
 	}
 
+	// we need to set product_name to support cloud-init
+	dmArgs = append(dmArgs, "-smbios", "type=1,product=OpenStack Compute")
```

Comment:
How is this openstack? Can it be any string or is there a set of standard ones?

Reply:
We should pass checks inside https://github.com/canonical/cloud-init/blob/a10838a621f2eef3f97959ae8ffb80bc2cd3b5fd/cloudinit/sources/DataSourceOpenStack.py#L233-L246. So, we should set appropriate product_name (or chassis_asset_tag) field.

Reply:
But that would imply that we claim to run neutron which means that the cloud init script might expect other openstack pieces to be set.

I think we'd just like to be able to pass things like the hostname and the set of network interfaces (one or more) as meta-data and have the cloud init scripts in the different Linux releases use that to configure. Maybe we also want to pass information about multiple virtual disks to automate the mounting of them (but I haven't looked at the disk support in the meta-data schemas).
It would be nice if we could use a schema where we don't have to pretend that EVE is a full openstack nor a full AWS infrastructure to pass that information.